jsk_pcl_ros/sample/sample_color_histogram_matcher.launch

```diff
@@ -0,0 +1,44 @@
+<launch>
```

Comment:
Can you also update `Sample` in `color_histogram_matcher.md`?

Reply:
I updated document.